### DISCUSSION TITLE
Enquiry regrading Gsoc

### DISCUSSION THREAD

**Discussion by Rishiguin** [learningequality/kolibri#10177] [Q&A] [answered]
Hi guys, I am unable to get in contact with the folks from Learning Equality to discuss about GSoC projects.

The only means of communication listed on the GSoC page is an email (gsoc@learningequality.org) and the Learning Equality Medium blog.
In the [contributor guidance ](https://docs.google.com/document/d/1-qmZ2CBl_WSw2s6oxH417y03Gudx8bFMSKG7IreB6JY/edit) and [IdeasList](https://docs.google.com/document/d/1P86snaquqrI03b9sFQhXBft3XXcXI5qUzlrxPQhXm8g/edit#heading=h.bvgjhf199bpc) there is a mention of a #gsoc-2023 slack channel but there is not any link to it so I sent an email at gsoc@learningequality.org.

I sent it 3 days ago and haven't received any response. Would be grateful if you guys could look into it.
My email : maharshiguin1000@gmail.com
![image](https://user-images.githubusercontent.com/47325298/222364800-be758cd9-958b-4c68-a559-a3195de699c6.png)

**Comment by radinamatic** [comment, marked as answer]
Hello @Rishiguin, thank you for your interest in contributing to Learning Equality through GSoC!

We've sent the Slack invite to you email, do check the pinned documents there as they contain all the information necessary for preparing your proposal. If after reading you have specific questions, don't hesitate to ask in the channel.

**Reply by Rishiguin** [reply]
Got the invite @radinamatic. Thank you very much

**Reply by josephinoo** [reply]
How can I get an invitation to join the GSoC 2023 Slack channel? I'm interested in working on the project "Adding a custom text color picker for Kolibri EPUB renderer". My email is [josdavilalvarez@gmail.com](mailto:josdavilalvarez@gmail.com). @radinamatic

**Reply by radinamatic** [reply]
Hello @josephinoo, check your email, you should have the Slack channel invite waiting 🙂 

Remember to read the pinned documents in the channel as they contain all the information necessary for preparing your proposal. For any other specific questions, do ask in the channel!

**Reply by Rohitrky2021** [reply]
How can I get an invitation to join the GSoC 2023 Slack channel? I'm interested in working on the project "Adding a custom text color picker for Kolibri EPUB renderer". My email is (rohit.21ug2037@iiitranchi.ac.in). @radinamatic

**Comment by Babithakumari** [comment]
Hey, I wanted to join the slack channel, it'd be great if anyone can share the link.
This is my mail id : babithakumari120@gmail.com
@radinamatic

**Reply by Rohitrky2021** [reply]
Plz provide the slack link so that i can review my proposal from mentors

**Comment by Rohitrky2021** [comment]
@Rishiguin @radinamatic  Plz provide the slack link so that i can review my proposal from mentors

**Comment by radinamatic** [comment]
@Rohitrky2021 and @Babithakumari, check your email, you should have the Slack channel invite waiting.

Please remember to read the pinned documents in the channel as they contain all the information necessary for preparing your proposal. For any other specific questions, do ask in the channel!

**Reply by Babithakumari** [reply]
Received the link.Thankyou @radinamatic

**Comment by manavagr1108** [comment]
Hey @radinamatic, I would like to contribute to Kolibri, can you please send the invite link for slack and other resource link to my gmail Id: [manavagr1108@gmail.com](mailto:manavagr1108@gmail.com). Thank you!

**Reply by manavagr1108** [reply]
I would like to work on the project idea `Facilitating H5P and SCORM import in Kolibri Studio`, have started writing my proposal, so please do send me the invite link for the slack channel so that I improve the proposal quality. Thank you!
@radinamatic

**Reply by manavagr1108** [reply]
Thank you @radinamatic, I have recieved the invitation.
